src/interface/update.cpp

```diff
@@ -154,79 +154,73 @@ TaskStatus SparseDealloc(MeshData<Real> *md) {
 
   const int num_blocks = pack.GetDim(5);
   const int num_vars = pack.GetDim(4);
-  ParArray2D<bool> is_zero("IsZero", num_blocks, num_vars);
 
-  const Real threshold = Globals::sparse_config.deallocation_threshold;
+  auto control_vars = md->GetMeshPointer()->resolved_packages->GetControlVariables();
 
+  const auto tup = SparsePack<>::Get(md, control_vars, {Metadata::Sparse});
+  auto pack2 = std::get<0>(tup);
+  auto pack2Idx = std::get<1>(tup);
+  for (int b = 0; b < num_blocks; ++b) {
+    for (auto &pair : pack2Idx) {
+      int lo = pack2.GetLowerBoundHost(b, PackIdx(pack2Idx[pair.first]));
+      int hi = pack2.GetUpperBoundHost(b, PackIdx(pack2Idx[pair.first]));
+    }
+  }
```

Comment:
It is there because I was testing the host functions. I have removed it, as well as some other code that was left over from using the old style packs.